ee/urls.py

```diff
@@ -2,15 +2,14 @@
 
 from django.conf import settings
 from django.contrib import admin
-from django.contrib.admin.sites import NotRegistered
+from django.contrib.admin.sites import NotRegistered  # type: ignore
```

Comment:
I see what you did 😄 Nitpick: `# type: ignore[import]` (it's pretty obvious though, free to ignore)